src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -108,7 +108,7 @@ const Allergies = props => {
             inline: true,
           },
           {
-            title: 'Observed or reported',
+            title: 'Signs and symptoms',
             value: item.observedOrReported,
             inline: true,
           },
```

Comment:
It looks like this changes the label for `item.observedOrReported` to "Signs and symptoms"
Instead, it Observed/Reported has gone away, and the label for "Reaction" should be changed to "Signs and symptoms".